config/cache.php

```diff
@@ -15,7 +15,7 @@
     |
     */
 
-    'default' => env('CACHE_DRIVER', 'file'),
+    'default' => env('CACHE_DRIVER', 'redis'),
```

Comment:
Switching to the redis cache is necessary for using the `->onOneServer()` functionality in Kernel.php. One of the big problems with scheduling before was that we'd often times end up with quadruplicate generated reports, and that's because the task was running on all four of our production nodes (two web, two worker). This should prevent that happening any longer, although it'll need testing on staging. We were also getting quadruplicate reports generated, which doesn't have a ton of consequences, but does soak up a bunch of extra resources for no good reason.